### ISSUE TITLE
Generate UMD modules without deps?

### ISSUE THREAD

**Issue by matthewwithanm** [substack/node-browserify#619]
Is it possible to generate UMD modules without the dependencies bundled and without having to list out every dependency to exclude?

My use case is that I have a multi-module library. Some of the modules in the library require others. I'd like to create UMD versions of each module (but not make them standalone) so that end users can use RequireJS, etc, and only require the modules they need to use.

In other words, I'm looking for a 1:1 conversion of node/CommonJS modules to UMD modules.

Thanks!
